knowledge-base/system-methodaccessexception.md

```diff
@@ -0,0 +1,46 @@
+---
+title: System.MethodAccessException - Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed
+description: System.MethodAccessException - Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed.
+type: how-to
+page_title: System.MethodAccessException -  Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed
+slug: system-methodaccessexception
+res_type: kb
+---
+
```

Comment:
Environment table

Reply:
The KB looks like a troubleshooting article. Please follow the KB standards and add all missing parts, like environment table, error message, cause, steps to reproduce, solution, etc..
https://wiki.progress.com/display/DTS/Knowledge+Base+Content+Standards#KnowledgeBaseContentStandards-ComponentsofTroubleshootingArticles